cgmes-gl/cgmes-gl-conversion/src/main/java/com/powsybl/cgmes/gl/conversion/LinePositionExporter.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.cgmes.gl.conversion;
+
+import java.util.Objects;
+
+import com.powsybl.cgmes.iidm.extensions.gl.LinePosition;
+import com.powsybl.iidm.network.DanglingLine;
+import com.powsybl.iidm.network.Line;
+import com.powsybl.triplestore.api.TripleStore;
+
+/**
+ *
+ * @author Massimo Ferraro <massimo.ferraro@techrain.eu>
+ */
+public class LinePositionExporter extends AbstractPositionExporter {
+
+    public LinePositionExporter(TripleStore tripleStore, ExportContext context) {
+        super(tripleStore, context);
+    }
+
+    public void exportPosition(Line line) {
+        Objects.requireNonNull(line);
+        LinePosition<Line> linePosition = line.getExtension(LinePosition.class);
+        exportPosition(line.getId(), line.getName(), linePosition);
+    }
+
+    public void exportPosition(DanglingLine danglingLine) {
+        Objects.requireNonNull(danglingLine);
+        LinePosition<DanglingLine> linePosition = danglingLine.getExtension(LinePosition.class);
```

Comment:
It will be a common case to not have an extension for each of the substation/line. In that case the code throws a NPE.

Reply:
fixed